docs/integrations/asgi.md

```diff
@@ -56,6 +56,23 @@ called "example".
 Then we use the context in a resolver, the resolver will return "1" in this
 case.
 
+### Setting response headers
+
+It is possible to use `get_context` to set response headers. A common use case might be cookie-based user authentication,
+where your login mutation resolver needs to respond with e.g. `Set-Cookie: token=secret-token`.
+
+This is possible by updating the response dict contained inside of the `Info` object.
+
+```python3
+@strawberry.type
+class Mutation:
+    @strawberry.field
+    def login(self, info: Info) -> bool:
```

Comment:
```suggestion
    def login(self, info) -> bool:
```

We don't have a well typed info object yet and adding it might cause confusion.

Reply:
we do have it :D https://github.com/strawberry-graphql/strawberry/releases/tag/0.45.4

Reply:
Oh my mistake! Ignore me then